### PR TITLE
[COUNTING] Task

### PR DESCRIPTION
**Pull request by eugene123tw**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by eugene123tw** [comment]
run ote_sdk tests

File: ote_sdk/ote_sdk/tests/entities/test_label.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ def test_domain(self):
         """
         domain = Domain
         assert len(domain) == 6
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_label.py]: remove this line

**Comment by eugene123tw** [comment]
run ote_sdk tests

File: ote_sdk/ote_sdk/usecases/exportable_code/prediction_to_annotation_converter.py [hunk 2]
```diff
@@ -151,8 +151,6 @@ def create_converter(
         converter = ClassificationToAnnotationConverter(labels)
     elif converter_type == Domain.ANOMALY_CLASSIFICATION:
         converter = AnomalyClassificationToAnnotationConverter(labels)
-    elif converter_type == Domain.COUNTING:
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/usecases/exportable_code/prediction_to_annotation_converter.py]: don't remove

**Comment by eugene123tw** [comment]
run ote_sdk tests

**Comment by eugene123tw** [comment]
run ote_sdk tests

**Comment by eugene123tw** [comment]
run ote_sdk tests

**Comment by eugene123tw** [comment]
run ote_sdk tests

**Comment by eugene123tw** [comment]
run ote_sdk tests

**Comment by eugene123tw** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
please merge `develop` into your branch

**Comment by eugene123tw** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
please merge develop into your branch

**Comment by eugene123tw** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Pull request merged**